### ISSUE TITLE
Applying the jerk factor to the flowrate

### ISSUE THREAD

**Issue by gedeonlambert** [repetier/Repetier-Firmware#76]
Hi,

While playing with the jerk settings a small question popped.
To prevent bubbling why the jerk factor isn't applied to the flowrate?

Since it's a pretty obvious thing I bet there's a pretty obvious explanation.

From what I understand, the jerk is calculated as a factor in `computeMaxJunctionSpeed()`.
Why not apply that factor the the flowrate?
Or better, give an option to do so.

**Comment by turpinator** [comment]
Old issue, but relevant.  Is there a way to set a max jerk?  (even if its just modifying the code one off)

Ive picked up zesty nimble which runs a worm gear to the hobb, and at ~3000 steps/mm, it cant handle large instantaneous speed changes.  Im sure im not the only one using repetier firmware, so this will probably get asked many times in the future.

**Comment by repetier** [comment]
Code above is still valid, so extruder start speed is equal to extruder jerk.

**Comment by turpinator** [comment]
So lets say i want an ejerk of 1...
its as simple as replacing the extruder start speed with 1, right?
[https://github.com/repetier/Repetier-Firmware/blob/9ac153211cf94797fdad4381087686ebaa2f3e91/src/ArduinoAVR/Repetier/motion.cpp#L859](url)
`    //if(eJerk > Extruder::current->maxStartFeedrate) //comment out this line, cause ejerk is almost always guaranteed to be greater than 1... right?
        factor = RMath::min(factor, 1.0 / eJerk); //and just replace the call to maxstartfeedrate with 1?`

cpp isnt my deal at all, so im really just poking around, but it should be as simple as that, right?

**Comment by repetier** [comment]
Yes just set start speed 1, but do not comment the line that tests it. That would disable eJerk completely or damage computation.
